pc_topic: "/velodyne_points"
map_topic: "/projected_map"
marker_topic: "occupied_cells"
reset_topic: "reset"
world_frame: "map"
robot_frame: "map"
width: 225
height: 25
ignore_top_rows: 4
resolution: 0.03225
z_minimum: 0.0275
run_frequency: 10.0
translate_distance: 0.5
sensing_radius: 0.0
